pkgs/development/python-modules/requests-wsgi-adapter/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "requests-wsgi-adapter";
+  version = "0.4.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-WncJ6Qq/SdGB9sMqo3eUU39yXeD23UI2K8jIyQgSyHg=";
+  };
+
+  propagatedBuildInputs = [
+    requests
+  ];
+
+  # tests are not contained in pypi-release
```

Comment:
Does it make sense to use `fetchFromGitHub` so that we can run the tests?

Reply:
sadly the repository doesn't contain an releases/tags so then we would have to refer to a specific commit